<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>271</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listWidgetFriend">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>271</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget::item:selected
{
       background: rgba(0,0,0,0);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>261</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditAdd"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddFriend">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelFriendCall">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHangUp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FF0000;
    border: 1px solid white;
    border-radius: 20px;
}

QPushButton:hover {
    background: qradialgradient(
        cx: 0.5, cy: -0.5, fx: 0.5, fy: -0.5,
        radius: 1.2, stop: 0 #fff, stop: 1 #FF0000
        );
}

QPushButton:pressed {
    border-style: inset;
    background-color: #FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="pushButtonCallAccept">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #6EFF33;
    border: 1px solid white;
    border-radius: 20px;
}

QPushButton:hover {
    background: qradialgradient(
        cx: 0.5, cy: -0.5, fx: 0.5, fy: -0.5,
        radius: 1.2, stop: 0 #fff, stop: 1 #6EFF33
        );
}

QPushButton:pressed {
    border-style: inset;
    background-color: #6EFF33;
}

QPushButton:disabled {
background-color: gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCallDeny">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FF0000;
    border: 1px solid white;
    border-radius: 20px;
}

QPushButton:hover {
    background: qradialgradient(
        cx: 0.5, cy: -0.5, fx: 0.5, fy: -0.5,
        radius: 1.2, stop: 0 #fff, stop: 1 #FF0000
        );
}

QPushButton:pressed {
    border-style: inset;
    background-color: #FF0000;
}

QPushButton:disabled {
background-color: gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTalk">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>439</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: grey;
    border: 1px solid white;
    border-radius: 20px;
}

QPushButton:hover {
    background: qradialgradient(
        cx: 0.5, cy: -0.5, fx: 0.5, fy: -0.5,
        radius: 1.2, stop: 0 #fff, stop: 1 grey
        );
}

QPushButton:pressed {
    border-style: inset;
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_talk">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TALK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menubabel">
    <property name="title">
     <string>babel</string>
    </property>
   </widget>
   <addaction name="menubabel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
